src/pages/lesson-plans/lesson-06.html

```diff
@@ -337,26 +337,26 @@ <h2 class="main">Integrated Lesson # 6</h2>
 <p>1. Introduction/Review (5 minutes)</p>
 <ol>
 <li>
-<step><p>Review rules quickly.</p>
+<step [stepsContext]="steps" [highlightedStepId]="suggestedStepId" [currentQuestionId]="gettingResponsesFor"><p>Review rules quickly.</p>
```

Comment:
Here is the start of a Step component -- it's also just a draft, but I've thought it through a bit more.  This is an example of a step which is "just" a step to be toggled (not a question with responses to be gathered)

Reply:
So as I start talking about implementation details, they're just what I envisioned...  but that doesn't mean I've thought it all the way through.  Feel free to change or whatever as needed.